libs/heattracesidesheet/src/lib/utils-sidesheet/useGetEleNetwork.ts

```diff
@@ -0,0 +1,43 @@
+import { ElectricalNetwork, useContextId, useHttpClient } from '@cc-components/shared';
+import { useQuery } from '@tanstack/react-query';
+import { HeatTrace } from '@cc-components/heattraceshared';
+
+export const useGetEleNetwork = (ht: HeatTrace | null) => {
+  const client = useHttpClient();
+  const contextId = useContextId();
+
+  if (!ht) {
+    throw new Error(`Failed to resolve the heat trace`);
+  }
+
+  const { data, isLoading, error } = useQuery<ElectricalNetwork | null>({
+    queryKey: [ht.heatTraceCableNo, ht.facility, ht.project],
+    queryFn: async ({ signal }) => {
+      const res = await client.fetch(
+        `api/contexts/${contextId}/electrical/electrical-network/${encodeURIComponent(
+          ht.heatTraceCableNo

```

Comment:
All the other resources for heat-trace seems to be using the `heatTraceCableId` field why is this one using the `heatTraceCableNo` field?